client/lib/plans/constants.js

```diff
@@ -94,11 +130,86 @@ export const plansList = {
 			FEATURE_GOOGLE_AD_CREDITS,
 			WORDADS_INSTANT,
 			FEATURE_GOOGLE_ANALYTICS
-		]
+		],
+		getBillingTimeFrame: () => i18n.translate( 'per month, billed yearly' )
 	},
 
-	[ PLAN_JETPACK_FREE ]: {},
-	[ PLAN_JETPACK_BUSINESS ]: {}
+	[ PLAN_JETPACK_FREE ]: {
+		getTitle: () => i18n.translate( 'Free' ),
+		getProductId: () => 2002,
+		getDescription: () => '',
+		getFeatures: () => [],
+		getBillingTimeFrame: () => i18n.translate( 'for life' )
+	},
+	[ PLAN_JETPACK_PREMIUM ]: {
+		getTitle: () => i18n.translate( 'Premium' ),
+		getProductId: () => 2000,
+		getDescription: () => i18n.translate( 'All the features you need to keep your site’s content backed up and secure, as well as spam-free.' ),
+		getFeatures: () => [
```

Comment:
I added couple of checkout-related attributes in #6207 , I think they should play nicely